x-pack/plugins/rule_registry/server/event_log/log/utils/mapping_from_field_map.ts

```diff
@@ -11,7 +11,7 @@ import { IndexMappings } from '../../elasticsearch';
 
 export function mappingFromFieldMap(fieldMap: FieldMap): IndexMappings {
   const mappings = {
-    dynamic: 'strict' as const,
+    dynamic: true,
```

Comment:
Temporary workaround to unblock work, will not be merged. Seems to be newly introduced mapping issues related to the security solution rules that are being worked on.